### PR TITLE
[author][new] Add across-tabs w/ npm auto-update via single package.json

### PR DESCRIPTION
**Pull request by softvar**
## Adding [across-tabs](https://github.com/wingify/across-tabs) library

Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

# Profile of the lib
 * Git repository (required): https://github.com/wingify/across-tabs
 * Official website (optional, not the repository): http://engineering.wingify.com/across-tabs/introduction.html
 * NPM package url (optional): https://www.npmjs.com/package/across-tabs
 * License and its reference: [MIT](https://github.com/wingify/across-tabs/blob/master/LICENSE)
 * GitHub / Bitbucket popularity (required):
   - Count of watchers: 40
   - Count of stars: 774
   - Count of forks: 18
 * NPM download stats (optional):
   - Downloads in the last day: 0
   - Downloads in the last week: 5
   - Downloads in the last month: 178

# Essential checklist
 * [x] I'm the author of this library
   * [x] I would like to add link to the page of this library on CDNJS on website / readme
 * [x] This lib was not found on cdnjs repo
 * [x] No already exist / duplicated issue and PR
 * [x] The lib has notable popularity
   * [x] More than 100 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [ ] More than 500 downloads stats per month on npm registry
 * [x] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [x] Has valid tags for each versions (for git auto-update)
 * [x] Auto-update setup
 * [x] Auto-update target/source is valid.
 * [x] Auto-update filemap is correct.

# Git commit checklist
 * [x] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
 * [x] The parent of the commit(s) in the PR is not old than 3 days.
 * [x] Pull request is sending from a non-master branch with meaningful name.
 * [x] Separate unrelated changes into different commits.
 * [x] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [x] Close corresponding issue in commit message
 * [x] Mention related issue(s), people in commit message, comment.

### CODE REVIEW

**Comment by softvar** [comment]
@maruilian11 @PeterDaveHello Please have a look

**Comment by softvar** [comment]
@sufuf3 Since cdnjs repo is too big to clone, so I have used Github GUI to open this PR. Otherwise, I always use command line interface.

Could you please do the `squashing` part on my behalf? I seriously don't want to clone this repo(~4GB). Or please tell me if there's something else I can do.

I thought, filling the following was optional.
```
 * NPM download stats (optional):
   - Downloads in the last day:
   - Downloads in the last week:
   - Downloads in the last month:
```
I have filled it now. Please check.

**Comment by sufuf3** [comment]
@softvar you can use `sparse-checkout` which lets you check out only the files you want. 
Here is the tutorial: https://github.com/cdnjs/cdnjs/blob/master/documents/sparseCheckout.md

Would you want to try?
Or let me help you do the `squashing`?

Thank you.

**Comment by softvar** [comment]
@sufuf3 Sure, I'm trying `sparse-checkout` as mentioned in the file. It still taking time because of `.git` folder. But I learned a new git trick and I'm happy about it.

Will let you know the status soon.

Thanks for helping.

File: ajax/libs/across-tabs/package.json [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+  },
+  "keywords": [
+    "across-tabs",
+    "browser tabs",
+    "tabs communication",
+    "tabs postmessage",
+    "postmessage",
+    "webpack",
+    "es6",
+    "umd"
```
Comment (extend1994) [inline ajax/libs/across-tabs/package.json]: One difference from the original `package.json`: `commonjs`. Is there any reason to delete it? Thanks

**Comment by sufuf3** [comment]
@softvar great!  Feel free to tag us, if you have any request.
Thank you for your contribution.

**Comment by softvar** [comment]
The task is taking too long.
Below is the screenshot of my terminal running commands. Please have a look.

screenshot 2017-04-17 14 10 17

Can you do the squashing part for me? I can't help myself.

**Comment by sufuf3** [comment]
@softvar OK I'll help you to squash the commit.
And would you want to add `commonjs` into `keywords`? 
https://github.com/wingify/across-tabs/blob/master/package.json#L54

If you want to add it, maybe I'll squash after you add it.
Or I can help you add it and squash, if you want.

File: ajax/libs/across-tabs/package.json (hunk 1 above)
Comment (softvar) [inline ajax/libs/across-tabs/package.json]: No specific reason, I was planning to remove it from the original repo too. But for now, I have added it again.

**Comment by softvar** [comment]
@sufuf3 I have added `commonjs` back to the list of keywords. Please have a look.
And please squash the commits.

Thanks.

**Comment by PeterDaveHello** [comment]
@softvar looks good, I'll help squash the commits into only one commit and use a better commit message :)

**Comment by softvar** [comment]
Thanks, @PeterDaveHello, @sufuf3 and others.
It's good to see the library hosted on [cdnjs.com](https://cdnjs.com).

I have added the cdnjs badge too in the repo's [README file](https://github.com/wingify/across-tabs/blob/master/README.md)

**Comment by PeterDaveHello** [comment]
@softvar :+1:

**Comment by PeterDaveHello** [comment]
@softvar actually I just sent it about a hour ago: https://github.com/wingify/across-tabs/pull/4

**Pull request merged**